go-app-ussd_public_rapidpro.js

```diff
@@ -279,7 +279,10 @@ go.app = function() {
                         return self.states.create("state_public_subscription");
                     } else if(_.inRange(_.get(contact, "fields.prebirth_messaging"), 1, 7)) {
                         return self.states.create("state_clinic_subscription");
-                    } else {
+                    } else if(_.toUpper(_.get(contact, "fields.postbirth_messaging")) === "TRUE"){
+                        return self.states.create("state_postbirth_subscription");
```

Comment:
I think this can probably just go to `state_clinic_subscription`, it doesn't mention prebirth

Reply:
Cool.